# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../thunderbird/powerpc64.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- mozilla/security/coreconf/Linux.mk.vanilla	2006-05-15 13:02:02.000000000 +0200
+++ mozilla/security/coreconf/Linux.mk	2006-05-15 13:02:29.000000000 +0200
@@ -60,6 +60,10 @@
 	OS_REL_CFLAGS	= -DLINUX1_2 -D_XOPEN_SOURCE
 	CPU_ARCH	= ppc
 else
+ifeq ($(OS_TEST),ppc64)
+        OS_REL_CFLAGS   = -DLINUX1_2 -D_XOPEN_SOURCE
+        CPU_ARCH        = ppc
+else
 ifeq ($(OS_TEST),alpha)
         OS_REL_CFLAGS   = -D_ALPHA_ -DLINUX1_2 -D_XOPEN_SOURCE
 	CPU_ARCH	= alpha
@@ -119,6 +123,7 @@
 endif
 endif
 endif
+endif
 
 
 LIBC_TAG		= _glibc
